### ISSUE TITLE
Unable to update the certificate.

### ISSUE THREAD

**Issue by apisit** [open-eid/qesteidutil#75]
Hi,
I am using ID-card utility version. `3.12.9.1261` and I have been trying to update the e-residency certificate but unable to do so.

Here is the response I received. 

<img width="573" alt="screen shot 2017-11-03 at 8 09 49 pm" src="https://user-images.githubusercontent.com/1012916/32371120-f8f7e476-c0d2-11e7-8443-5776f25a0662.png">

This is the info when I click `Details` button.

```
Connecting to "https://id-gw.eesti.ee/update/"
Proxy "" : 0 User ""
< {"cmd":"START","lang":"en","platform":"Mac OS 10.12.6 (64/x86_64)","version":"3.12.9.1261"}
```

Not really sure whether the server is really overloaded or just my machine. 
However, I just want to report it.

**Comment by vanjek** [comment]
Had the same issue but after trying dozen of times managed to get to the next screen

![23261874_10215109079748289_891289848_o](https://user-images.githubusercontent.com/3473899/32387956-65b4bdd0-c0c6-11e7-9777-307e9ac58964.png)

My initial thought was that it passed and that it should work now, but oh well...
Updating failed again, now with the red text and following details:

`
{"cmd":"DIALOG","text":"For updating certificates please close all programs which are interacting with smartcard (DigiDoc Client, DigiDoc Crypto, Chrome, Firefox, Safari, Internet Explorer...)<a href=\"http:\/\/www.id.ee\/?id=37548\">More info<\/a>.<br \/><br \/>After updating certificates it will no longer be possible to decrypt documents that were encrypted with the old certificate!<br \/><br \/>If the renewal of the certificates has failed, you can try again within 30 days. After that, you can only renew your ID-card certificates at any service center of the Police and Border Guard Board.<br \/><br \/><a href=\"https:\/\/www.politsei.ee\/dotAsset\/674066.pdf\">Conditions for Use of Certificates<\/a>.","session":"..."}
`

If I try to do it again, I am back to:

`Connecting to "https://id-gw.eesti.ee/update/"
Proxy "" : 0 User ""
< {"cmd":"START","lang":"en","platform":"Mac OS 10.13.1 (64/x86_64)","version":"3.12.9.1261"}`

After trying few more times, managed to pass the first step again, but now my ID-card certificate renewal has been suspended.

<img width="697" alt="screen shot 2017-11-03 at 6 49 40 pm" src="https://user-images.githubusercontent.com/3473899/32388415-dd2dce96-c0c7-11e7-93c9-166132541fda.png">

Slowly loosing my patience (:

**Comment by tynisr** [comment]
Hi

As of midnight Estonian local time all the vulnerable certificates have been suspended (meaning they can not be used until renewed). At the moment the online certificate renewal is available only for critical personel (like medical workers, rescue and security) and heavy everyday users (for example people using it daily for login).
We expect the online certificate renewal to be available again for the general public on Monday, but server overload can still be expected. Unless you need to use your electronic ID immediately next week you should not rush the update, the update will remain available until 31.03.2018. 
For further information please contact the e-recidency program. See also: http://id.ee/index.php?id=30610&read=38344

For support please contact with:
----------------------------------------------
Help centre : http://id.ee/index.php?id=30466
Helpline (+372) 677 3377 with English support
E-mail abi@id.ee
